install_config/topics/ocp_osp_deploy_instance.adoc

```diff
@@ -0,0 +1,271 @@
+==== Creating and Configuring the Deployment Instance
+
+The role of the deployment instance is to serve
+as a utility host for the deployment and management of {product-title}.
+
+===== Creating the Deployment Host Network and Router
+
+Prior to instance creation, an internal network and router
+must be created for communication with the deployment host. The 
+following commands create that network and router.
+
+[subs=+quotes]
+----
+$ source path/to/examplerc
+
+$ openstack network create *<deployment-net-name>*
+
+$ openstack subnet create --network *<deployment-net-name>* \
+  --subnet-range *<subnet_range>* \
+  --dns-nameserver *<dns-ip>* \
+  *<deployment-subnet-name>*
+
+$ openstack router create *<deployment-router-name>*
+
+$ openstack router set --external-gateway *<public-net-name>* *<deployment-router-name>*
+
+$ openstack router add subnet *<deployment-router-name>* *<deployment-subnet-name>*
+----
+
+===== Deploying the Deployment Instance
+
+With the network and security group created, deploy the instance.
+
+[subs=+quotes]
+----
+$ domain=*<domain>*
+$ netid1=$(openstack network show *<deployment-net-name>* -f value -c id)
+$ openstack server create \
+    --nic net-id=$netid1 \
+    --flavor *<flavor>* \
+    --image *<image>* \
+    --key-name *<keypair>* \
+    --security-group *<deployment-sg-name>* \
+    deployment.$domain
+----
+
+NOTE: If the `m1.small` flavor does not exist by default then use an existing
+flavor that meets the requirements of 1 vCPU and 2GB of RAM.
+
+===== Creating and Adding Floating IP to the Deployment Instance
+
+Once the deployment instance is created, a floating IP must be created and then
+allocated to the instance. The following shows an example
+
+[subs=+quotes]
+----
+$ source /path/to/examplerc
+$ openstack floating ip create *<public-network-name>*
```

Comment:
@rlopez133 
# openstack floating ip create public_network
Error while executing command: HttpException: Unknown error, {"NeutronError": {"message": "No more IP addresses available on network 5f347039-cd01-4cc9-b126-e6d7df65bf4a.", "type": "IpAddressGenerationFailure", "detail": ""}}

Reply:
This is 1 of 2 things, our environment ran out of floating IPs (likely) or your quota for the amount of floating IPs is exhausted.

Reply:
i wouldn't worry here as that was specific to our env but the step is correct.